sdk/containerregistry/azure-containerregistry/samples/sample_delete_images.py

```diff
@@ -35,7 +35,7 @@
 class DeleteImages(object):
     def __init__(self):
         load_dotenv(find_dotenv())
-        self.endpoint = os.environ.get("CONTAINERREGISTRY_ENDPOINT")
+        self.endpoint: str = os.environ.get("CONTAINERREGISTRY_ENDPOINT")  # type: ignore[assignment]
```

Comment:
Is endpoint optional for this sample? if not, we can do `os.environ["CONTAINERREGISTRY_ENDPOINT"]` and that should clear your type error.

edit: Comment applies generally to the samples. We should do our best to not use type: ignore in user code.

Reply:
Updated